src/ui/pages/AssetInfo/AssetResources.tsx

```diff
@@ -0,0 +1,365 @@
+        ['--button-background' as string]: 'var(--neutral-200)',
+        ['--button-background-hover' as string]: 'var(--neutral-300)',
+      }}
+    >
+      <HStack gap={4} alignItems="center">
+        <HStack gap={6} alignItems="center">
+          <img
+            src={network.icon_url}
+            alt={network.name}
+            width={24}
+            height={24}
+          />
+          <UIText kind="small/accent">
+            {middleTruncate({
+              value: address,
+              leadingLettersCount: 5,
+              trailingLettersCount: 4,
+            })}
+          </UIText>
+        </HStack>
+        {isSuccess ? (
+          <CheckIcon
+            style={{ color: 'var(--positive-500)', width: 20, height: 20 }}
+          />
+        ) : (
+          <CopyIcon
+            className="content-hover"
+            style={{ width: 20, height: 20 }}
+          />
+        )}
+      </HStack>
+    </Button>
+  );
+}
+
+function CopyAddressButton({ address }: { address: string }) {
+  const { handleCopy, isSuccess } = useCopyToClipboard({ text: address });
+
+  return (
+    <UnstyledButton
+      onClick={handleCopy}
+      aria-label="Copy token address"
+      className="parent-hover"
+      style={{
+        ['--parent-content-color' as string]: 'var(--neutral-400)',
+        ['--parent-hovered-content-color' as string]: 'var(--neutral-700)',
+        display: 'flex',
+      }}
+    >
+      {isSuccess ? (
+        <CheckIcon style={{ color: 'var(--positive-500)' }} />
+      ) : (
+        <CopyIcon className="content-hover" />
+      )}
+    </UnstyledButton>
+  );
+}
+
+function AssetImplementationsDialog({
+  implementations,
+}: {
+  implementations: { address: string | null; network: NetworkConfig }[];
+}) {
+  const { networks } = useNetworks();
+  return (
+    <VStack
+      gap={14}
+      style={{
+        ['--surface-background-color' as string]: 'var(--white)',
+        padding: '8px 16px 24px',
+      }}
+    >
+      <VStack gap={0}>
+        {implementations.map(({ address, network }) => (
+          <HStack
+            key={network.id}
+            gap={24}
+            justifyContent="space-between"
+            alignItems="center"
+            style={{ paddingBlock: 12 }}
+          >
+            <HStack gap={12} alignItems="center">
+              <img
+                src={network.icon_url}
+                alt={network.name}
+                width={36}
+                height={36}
+              />
+              <VStack gap={0}>
+                <UIText kind="body/accent">{network.name}</UIText>
+                {address ? (
+                  <UIText kind="small/regular" color="var(--neutral-500)">
+                    {middleTruncate({ value: address })}
+                  </UIText>
+                ) : null}
+              </VStack>
+            </HStack>
+            <HStack gap={0}>
+              {address ? <CopyAddressButton address={address} /> : null}
```

Comment:
missing `title` for accessibility

Reply:
Technically, CopyAddressButton, has aria-label.
But I can add title too

Reply:
if `title` is added, `aria-label` is no longer needed

Reply:
Ah, sorry, didn't get your question

For `CopyAddressButton` aria-label should be enough, because visual users recognize the common copy icon

But for the explorer name it's hard to understand from the icon the purpose of the link :)